languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/hunspell/spelling.txt

```diff
@@ -54916,5 +54916,13 @@ Industriegasekonzern/S
 Verkettungstermin/S
 Verkettungstermine/N
 fressbar/A
-Fr. #abk
-Zeitraffervideo/S
\ No newline at end of file
+Zeitraffervideo/S
+Motivwunsch
+Motivwünsche/N
+erbgroßherzoglich/A
+ErbStRG #abk
+Ferranti #name
+Freespace #name #eng
+Fry #name
```

Comment:
@Luisa-LT solche kurzen, eher seltenen Namen sollten wir lieber in die `ignore.txt` packen. Damit die nicht für Rechtschreibvorschläge genommen werden. Wörter in der `ignore.txt` werden nicht als Rechtschreibfehler unterstrichen und anders als in der `spelling.txt` nicht dazu verwendet, um Rechtschreibvorschläge für falsch geschriebene Wörter zu generieren.

Reply:
Danke für den Tipp und die Erklärung. Stimmt, ignore.txt ist dann hier die bessere Wahl! Werde es ändern.